test-case-reporting/README.md

```diff
@@ -1,31 +1,51 @@
 AMP Test Case Reporting Bot
 ==============
 
-[[description]]
+A Google Cloud app that stores the results of tests run by Travis. Used to track flaky tests.
 
 This app runs as a [[something]].
 
 Interface
 ---------
-[[what this listens for, what it does]]
+### API for Travis
+
+The App has the following API endpoints, which are to be triggered from Travis CI
+runs.
+
+* `POST /report`
+  * Accepts a JSON object representing a test result report from Travis. Has 3 fields: `build`, `job`, and `results`, representing build information, job information, and test run information respectively. For the full structure of the object, see [the type declaration module](types/test-case-reporting.d.ts).
 
 Setup
 -----
-[[no clue]]
+
+Follow these setup instructions to start developing for this App locally:
+
+1. Clone this repository and cd into the `test-case-reporting` directory
+2. `npm install`
+3. Create a new Google Cloud project and switch to it with `gcloud config set project [PROJECT_NAME]`.
+4. Add an SQL database to the Google Cloud project. Choose a name for the database and store it in the `DB_NAME` field of .env.
+5. Initialize an App Engine app with `gcloud app create'.
+6. Run a local instance of PostgreSQL, or use the
```

Comment:
How do you create the tables to initialize the Cloud SQL instance?

Reply:
Added instruction step for that.